target/linux/lantiq/image/Makefile

```diff
@@ -560,42 +624,52 @@ TARGET_DEVICES += zyxel_p-2812hnu-f1
 
 define Device/zyxel_p-2812hnu-f3
   $(Device/NAND)
+  DEVICE_VENDOR := ZyXEL
+  DEVICE_MODEL := P-2812HNU
+  DEVICE_VARIANT := F3
   BOARD_NAME := P2812HNUF3
   DEVICE_DTS := P2812HNUF3
-  DEVICE_TITLE := ZyXEL P-2812HNU-F3
   DEVICE_PACKAGES := kmod-rt2800-pci wpad-basic kmod-usb-dwc2
   SUPPORTED_DEVICES += P2812HNUF3
 endef
 TARGET_DEVICES += zyxel_p-2812hnu-f3
 
 define Device/arcadyan_arv7519rw22
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := ARV7519RW22
+  DEVICE_ALT0_VENDOR := Astoria Networks
+  DEVICE_ALT0_MODEL := Livebox
```

Comment:
Let me suggest another change for arv7519rw after reading @aparcar's pull request adding the new variables.
```
DEVICE_VENDOR := Arcadyan
DEVICE_MODEL := ARV7519RW22
DEVICE_ALT0_VENDOR := Orange
DEVICE_ALT0_MODEL := Livebox
DEVICE_ALT0_VARIANT := 2.1
DEVICE_ALT1_VENDOR := Astoria Networks
DEVICE_ALT1_MODEL := ARV7519RW22
```
If @adrianschmutzler suggestion about showing the first alternative in the build systems this will make the device appearing as Arcadyan ARV7519RW22 (Orange Livebox 2.1). The repeated model name is because Astoria uses the same model name as Arcadyan, but this way I think conforms @aparcar's pull request.

Reply:
Looks good to me. @adrianschmutzler how about you?

Reply:
I think the ALT0 as in the proposal is a good idea, I'm agnostic about the ALT1.

Reply:
ok, I've added @howl  proposed change.